backend/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fastapi
+uvicorn
+mistralai
```

Comment:
should we specify versions here? Feel like I've seen that somewhere to make sure people in the future get a working version? wdyt?

Reply:
From what I've read, it's good practice to specify a version. Can avoid dependency hell in future when some libraries have updated but others haven't.

It's also our way of saying: "this code has been proven to work successfully against these versions".

Reply:
I agree - I've added versions (& updated the readme as it had a mistake in it)